Fundamentals/Setup/Install/installing-nightly-builds.md

```diff
@@ -0,0 +1,101 @@
+
+You can use the package manager in Visual Studio to browse the available template versions. First open the package manager by selecting `Tools > NuGet Package Manager > Manage NuGet Packages For Solution...` from the top menu
+
+![Opening the Nuget Package Manager](images/VS/open-nuget-package-manager.jpg)
+
+Now the `NuGet` window will open, in the top right corner there is a dropdown `Package source`, in order to be able to see our nightly builds, we have to change this to the feed we've just added `Umbraco Nightly` in this case.
+
+![Select the nightly NuGet feed](images/VS/select-nuget-feed.jpg)
+
+Now that the correct feed is selected we're almost ready to search for the package and choose a version, however since these are marked as prerelease, we have to tick the checkbox `Include prerelase` otherwise these will be filtered out. Next search for `Umbraco.Templates`, and choose that package. Now we can click on the `Version` drop down and see the available nightly builds for both V9 and V10. It's important to note that this is the ***template***, not the Umbraco package itself, so it won't work to click install. Instead chose the applicable version and note down the version number
+
+![Find the version](images/VS/find-the-version.jpg)
+
+### Using Rider
+
+You can use the NuGet window in Rider to browse the available template versions. First open the window by clicking on `NuGet` on the bottom bar, next make sure the `Packages` tab is selected. To make things as easy as possible for ourselves click on the sources dropdown (3), and make sure that only our nightly feed is selected.
+
+![Choose the feed](images/Rider/choose-the-feed.jpg)
+
+Now that the correct feed is selected we're almost ready to search for the package and choose a version, however since these are marked as prerelease, we have to tick the checkbox `Prerelase` otherwise these will be filtered out. Next search for `Umbraco.Templates`, and choose that package. Now we can click on the `Version` drop down and see the available nightly builds for both V9 and V10. It's important to note that this is the ***template***, not the Umbraco package itself, so it won't work to click install. Instead chose the applicable version and note down the version number
+
+![Find the version](images/Rider/find-the-version.jpg)
+
+## Installing the nightly build
+
+Now that our feed is added and we now the exact version we're ready to install our template.
+
+Open up your command prompt of choice and execute the `dotnet new -i ` command using the latest, version, at the time of writing the latest preview build is `9.4.0-preview20220228.85007` so to install the newest nightly build I'd have to execute:
+
+```
+dotnet new -i Umbraco.Templates::9.4.0-preview20220228.85007
+```
+
+The name and the versions is seperated with two colons `::`, it's important that we specify the entire version including the `-preview20220228.85007`, otherwise the dotnet new command cannot find the package.
```

Comment:
```suggestion
The name and the versions are separated with two colons `::`, it's important that we specify the entire version including the `-preview20220228.85007`, otherwise the dotnet new command cannot find the package.
```